src/wrappers/AppInitializer.tsx

```diff
@@ -9,18 +9,21 @@ import { isLoginSessionInvalidSelector } from 'reduxStore/selectors/loginInfoSel
 import { initializeNetworkConfig } from 'reduxStore/slices/networkConfigSlice';
 import { CustomNetworkType, EnvironmentsEnum, IDappProvider } from 'types';
 import { logout } from 'utils/logout';
+import { setAutoLogoutCallbackUrl } from 'reduxStore/slices/loginInfoSlice';
 
 export interface AppInitializerPropsType {
   customNetworkConfig?: CustomNetworkType;
   children: any;
   externalProvider?: IDappProvider;
   environment: EnvironmentsEnum;
+  autoLogoutCallbackUrl?: string;
```

Comment:
it seems that in the future we might want to include more variables here. Instead of adding them one by one, it would be better to group them under an optional proprety called `dappConfig`

Reply:
agree